src/sdk/main/src/TokenUpdateNftsTransaction.cc

```diff
@@ -0,0 +1,137 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "TokenUpdateNftsTransaction.h"
+#include "impl/DurationConverter.h"
+#include "impl/Node.h"
+#include "impl/TimestampConverter.h"
+#include "impl/Utilities.h"
+
+#include <grpcpp/client_context.h>
+#include <proto/token_update_nfts.pb.h>
+#include <proto/transaction.pb.h>
+#include <proto/transaction_body.pb.h>
+#include <stdexcept>
+
+namespace Hedera
+{
+
```

Comment:
nit: remove the space between the `namespace` bracket and the first function definition